open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/OpenMetadataAPIMapper.java

```diff
@@ -1635,7 +1635,7 @@
     public static final int    REQUESTED_GA_STATUS_ORDINAL               = 0;
     public static final int    APPROVED_GA_STATUS_ORDINAL                = 1;
     public static final int    WAITING_GA_STATUS_ORDINAL                 = 2;
-    public static final int    IN_PROGRESS_GA_STATUS_ORDINAL             = 3;
+    public static final int    IN_PROGRESS_GA_STATUS_ORDINAL             = 4;
```

Comment:
I cant see the reason for this change

Reply:
The ordinal for GovernanceActionStatus.IN_PROGRESS is set to 4, not 3. The constant IN_PROGRESS_GA_STATUS_ORDINAL is used for logging, so without this change the audit log would say that the status is changed to UNKNOWN instead of IN_PROGRESS